packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -340,6 +396,11 @@ class CheckboxListTile extends StatelessWidget {
   /// inoperative.
   final bool? enabled;
 
+  /// {@macro flutter.material.checkbox.isError}
+  ///
+  /// defaults to false.
```

Comment:
```suggestion
  /// Defaults to false.
```

Reply:
Fixed!

Reply:
Still looks uncapitalized, maybe needs a push?

Reply:
Oh I might forget to commit the change. Thanks for catching this:)